src/pkg/utils/io.go

```diff
@@ -103,6 +103,12 @@ func ReplaceTextTemplate(path string, mappings map[string]string) {
 		text = bytes.ReplaceAll(text, []byte(template), []byte(value))
 	}
 
+	for old, new := range deprecations {
```

Comment:
Doing this here would never display the warning...  It would already be replaced

Reply:
thanks, multitasking fail.